src/js/views/check-ins/check-in/check-in_views.js

```diff
@@ -12,7 +12,7 @@ const ContextTrailView = View.extend({
   <a class="js-back check-in__context-link">
     {{fas "chevron-left"}}{{ @intl.checkIns.checkInViews.contextTrailView.backBtn }}
   </a>
-  {{fas "chevron-right"}}{{ patient.first_name }} {{ patient.last_name }} - {{formatHTMLMessage (intlGet "checkIns.checkInViews.contextTrailView.checkInCompleted") date=(formatMoment finished_at "LONG" inputFormat="YYYY-MM-DD") }}
+  {{fas "chevron-right"}}{{ patient.first_name }} {{ patient.last_name }} - {{formatHTMLMessage (intlGet "checkIns.checkInViews.contextTrailView.checkInCompleted") date=(formatMoment finishedTs "LONG") }}
```

Comment:
this would need a `utc=true` I believe otherwise this will load the date as if it is in the local timezone